tests/test_ditchthecarbs.py

```diff
@@ -11,22 +11,25 @@ def test_host(self):
 
     def test_canonical_url(self):
         self.assertEqual(
-            "https://www.ditchthecarbs.com/easy-keto-hamburger-buns-almond-flour/",
+            "https://thinlicious.com/easy-keto-hamburger-buns-almond-flour/",
             self.harvester_class.canonical_url(),
         )
 
     def test_title(self):
-        self.assertEqual(self.harvester_class.title(), "Keto Hamburger Buns Recipe")
+        self.assertEqual(
+            self.harvester_class.title(),
+            "Easy Keto Hamburger Buns Recipe (That Don't Fall Apart)",
+        )
 
     def test_author(self):
-        self.assertEqual(self.harvester_class.author(), "Libby Jenkinson")
+        self.assertEqual(self.harvester_class.author(), "Thinlicious")
```

Comment:
Hrm.  Something to look into here; Thinlicious do now host this recipe, but Libby should still be credited as the author.

Reply:
The only occurrence of Libby's name in the site is her comments back to some of the comments.

The schema ID returned is https://thinlicious.com/#/schema/person/3f3749d6ecf1ba36a7fefe4a97abdc70 which is associated with the name "Thinlicious". Seems like the rebrand striped the original author names and put all recipes under the site name.

Reply:
Okie doke.  We can't computationally determine credit then; this is a case where we can add a comment in the code to mention that the author was Libby, and to check for that again in future in case it becomes possible to determine again.